runners/flink/1.10/build.gradle

```diff
@@ -31,3 +35,19 @@ project.ext {
 
 // Load the main build script which contains all build logic.
 apply from: "$basePath/flink_runner.gradle"
+
+dependencies {
+  permitUnusedDeclared library.java.jackson_annotations
```

Comment:
The dependencies are set up in `flink_runner.gradle` using the versions in the `project.ext` block.

@mxm may have some comment or recommend someone for a comment.